Sources/Swift/Tools/SentryLogger.swift

```diff
@@ -0,0 +1,77 @@
+import Foundation
+
+@objc
+@objcMembers
+public final class SentryLogger: NSObject {
+    let hub: SentryHub
+    let dateProvider: SentryCurrentDateProvider
+    
+    @_spi(Private) public init(hub: SentryHub, dateProvider: SentryCurrentDateProvider) {
+        self.hub = hub
+        self.dateProvider = dateProvider
+        super.init()
+    }
+    
+    public func trace(_ body: String) {
```

Comment:
These mehtods are the user facing api, accessed by `SentrySDK.logger.trace(...)` etc.